apis/config/v1alpha2/defaults_test.go

```diff
@@ -15,46 +15,46 @@ func TestSetDefaults_Configuration(t *testing.T) {
 	}{
 		"defaulting namespace": {
 			original: &Configuration{
-				Namespace: "",
+				Namespace: nil,
 				InternalCertManagement: &InternalCertManagement{
 					Enable: pointer.Bool(false),
 				},
 			},
 			want: &Configuration{
-				Namespace: "kueue-system",
+				Namespace: pointer.String("kueue-system"),
 				InternalCertManagement: &InternalCertManagement{
 					Enable: pointer.Bool(false),
 				},
 			},
 		},
 		"defaulting InternalCertManagement": {
 			original: &Configuration{
-				Namespace: "kueue-tenant-a",
+				Namespace: pointer.String("kueue-tenant-a"),
 			},
 			want: &Configuration{
-				Namespace: "kueue-tenant-a",
+				Namespace: pointer.String("kueue-tenant-a"),
 				InternalCertManagement: &InternalCertManagement{
 					Enable:      pointer.Bool(true),
-					ServiceName: "kueue-webhook-service",
-					SecretName:  "kueue-webhook-server-cert",
+					ServiceName: pointer.String("kueue-webhook-service"),
```

Comment:
It makes sense.
Since we will sometimes create struct `InternalCertManagement` in `defaults_test.go` and `main_test.go`, I will add a Wrapper for InternalCertManagement in `testing` pkg, same as other wrappers.

Reply:
Done.